api/xcfun.h

```diff
@@ -128,10 +128,20 @@ const char * xc_describe_long(const char * name);
 XCFun_API int xc_is_gga(xc_functional fun);
 XCFun_API int xc_is_metagga(xc_functional fun);
 
-int xc_set_fromstring(xc_functional fun, const char * str); // Defines a functional
+XCFun_API int xc_set_fromstring(xc_functional fun, const char * str); // Defines a functional
                                                             // from a string on the
                                                             // form "fun[=value]"
 
+XCFun_API int xc_user_eval_setup(xc_functional fun,
```

Comment:
Also here we could benefit from having these comments as Doxygen documentation strings to the function itself. Example:
```
/*! \brief Creates a new PCM context object
 *  \param[in] input_reading input processing strategy
 *  \param[in] nr_nuclei     number of atoms in the molecule
 *  \param[in] charges       atomic charges
 *  \param[in] coordinates   atomic coordinates
 *  \param[in] symmetry_info molecular point group information
 *  \param[in] host_input    input to the module, as read by the host
 *  \param[in] writer        flush-to-host function
 *
 *  The molecular point group information is passed as an array
 *  of 4 integers: number of generators, first, second and third generator
 *  respectively. Generators map to integers as in table :ref:`symmetry-ops`
 */
PCMSolver_API pcmsolver_context_t * pcmsolver_new(pcmsolver_reader_t input_reading,
                                                  int nr_nuclei,
                                                  double charges[],
                                                  double coordinates[],
                                                  int symmetry_info[],
                                                  struct PCMInput * host_input,
                                                  HostWriter writer);
```

Reply:
The whole xcfun would benefit from a sensible doxygen documentation, but I believe that should be considered in a different occasion.